sql/schema.go

```diff
@@ -0,0 +1,202 @@
+package sql
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	godiffpatch "github.com/sourcegraph/go-diff-patch"
+	"go.uber.org/zap"
+)
+
+const (
+	SchemaPath        = "schema/schema.sql"
+	UpdatedSchemaPath = "schema/schema.sql.updated"
+)
+
+// LoadSchema loads the schema embedded in the executable.
+func LoadSchema(fsys fs.FS, migrations []Migration) (*Schema, error) {
```

Comment:
This function isn't actually needed (see my other comments).

Reply:
Removed